src/App.js

```diff
@@ -1,23 +1,76 @@
-import logo from './logo.svg';
-import './App.css';
+import React, { useEffect, useState } from "react";
+import htmlContent from "./content.json";
+import styles from "./App.module.scss";
+import Header from "../src/component/header/header";
+import Footer from "../src/component/footer/footer";
+import Main from "../src/component/main/main";
 
 function App() {
+  const [htmlHeader, setHtmlHeader] = useState("");
+  const [contentDescription, setContentDescription] = useState("");
+  const [contentIndex, setContentIndex] = useState(0);
+  const [nextPageLink, setNextPageLink] = useState("");
+  const [previousPageLink, setPreviousPageLink] = useState("");
+
+  useEffect(() => {
+    const initialContentIndex = 0;
+    setHtmlHeader(htmlContent.title);
+    setContentDescription(htmlContent.content[initialContentIndex].description);
```

Comment:
setContentDescription(htmlContent?.content?.[initialContentIndex]?.description);

Supposed you can't get htmlContent from the json file.

Reply:
Change other content inside this useEffect  as well.